akka-cluster-sharding-typed/src/multi-jvm/scala/akka/cluster/sharding/typed/MultiDcClusterShardingSpec.scala

```diff
@@ -0,0 +1,107 @@
+package akka.cluster.sharding.typed
```

Comment:
header? does not add here automatically? consider fixing the plugin config to do so

Reply:
I fixed the plugin but it is a lot of changes. I'll push that as a diff PR to keep this one clean.